compat/vcbuild/vcpkg_install.bat

```diff
@@ -34,6 +34,12 @@ REM ================================================================
 	@FOR /F "delims=" %%D IN ("%~dp0") DO @SET cwd=%%~fD
 	cd %cwd%
 
+	SET arch=x64-windows
+
+REM is vcpkg package installed and complied \\compat\\vcbuild\\vcpkg\\installed\\\$(VCPKGArch)\\include\\openssl\\ssl.h
+	dir vcpkg\installed\%arch%\include\openssl\ssl.h >nul 2>nul && GOTO :check_for_updates
+
+REM is vcpkg package manager already cloned
```

Comment:
Please fix the indentation.

Reply:
Local REM indentation is single space

Reply:
Look at it on GitHub. There is a difference between the indentation of all your `REM`s vs the `IF EXIST` line. It's totally jagged.